packages/dx-react-grid/docs/reference/table-summary-row.md

```diff
@@ -0,0 +1,125 @@
+# TableSummaryRow Plugin Reference
+
+A plugin that renders table rows that display total, group, and tree summary.
```

Comment:
A plugin that renders table rows that display total, group, and tree summary -> A plugin that renders table rows that display **a** total, group, and tree summary.